### PR TITLE
GCP VM Instance Stop Experiment Docs

### PR DESCRIPTION
**Pull request by neelanjan00**
Added experiment documentation and added sidebar id

### CODE REVIEW

File: docs/gcp-vm-instance-stop.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+  <tr>
+    <td> SEQUENCE </td>
+    <td> It defines sequence of chaos execution for multiple instance </td>
+    <td> Optional </td>
+    <td> Default value: parallel. Supported: serial, parallel </td>
+  </tr>  
+  <tr> 
+    <td> GCP_PROJECT_ID </td>
+    <td> GCP project ID to which the VM instances belong </td>
+    <td> Mandatory </td>
```
Comment (ispeakc0de) [inline docs/gcp-vm-instance-stop.md]: Can we move mandatory fields at the beginning of the list? followed by the optional fields

Comment (neelanjan00) [reply docs/gcp-vm-instance-stop.md]: Sure. Done.

File: docs/gcp-vm-instance-stop.md [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+  </tr>
+  <tr> 
+    <td> VM_INSTANCE_NAMES </td>
+    <td> Name of target VM instances </td>
+    <td> Mandatory </td>
+    <td> Multiple instance names can be provided as instance1,instance2,... </td>
+  </tr>
+  <tr>
+    <td> INSTANCE_ZONES </td>
+    <td> The zones of the target VM instaces </td>
```
Comment (ispeakc0de) [inline docs/gcp-vm-instance-stop.md]:
```suggestion
    <td> The zones of the target VM instances </td>
```

File: website/sidebars.json [hunk 3]
```diff
@@ -64,7 +64,8 @@
           "node-io-stress",
           "node-restart",
           "pod-cpu-hog-exec",
-          "pod-memory-hog-exec"          
+          "pod-memory-hog-exec",
+          "gcp-vm-instance-delete"
```
Comment (ispeakc0de) [inline website/sidebars.json]: we can remove this

File: website/sidebars.json [hunk 4]
```diff
@@ -78,6 +79,13 @@
           "ec2-terminate-by-tag"
         ]
       },
+      {
+        "type": "subcategory",
+        "label": "GCP",
+        "ids": [
+          "gcp-vm-instance-delete"
```
Comment (ispeakc0de) [inline website/sidebars.json]:
```suggestion
          "gcp-vm-instance-stop"
```

File: docs/gcp-vm-instance-stop.md [hunk 5]
```diff
@@ -0,0 +1,269 @@
...
+
+-   VM instance is healthy before chaos injection.
+
+## Exit-Criteria
+
+-   VM instance is healthy post chaos injection.
+
+## Details
+
+-   Causes termination of a GCP VM instance by instance name or list of instance names before bringing it back to the running state after the specified chaos duration.
```
Comment (uditgaurav) [inline docs/gcp-vm-instance-stop.md]:
```suggestion
-   Causes poweroff of a GCP VM instance by instance name or list of instance names before bringing it back to the running state after the specified chaos duration. 
```

File: docs/gcp-vm-instance-stop.md (hunk 2 above)
Comment (neelanjan00) [reply docs/gcp-vm-instance-stop.md]: Resolved

File: website/sidebars.json (hunk 3 above)
Comment (neelanjan00) [reply website/sidebars.json]: Fixed

File: website/sidebars.json (hunk 4 above)
Comment (neelanjan00) [reply website/sidebars.json]: fixed

File: docs/gcp-vm-instance-stop.md [hunk 6]
```diff
@@ -0,0 +1,269 @@
...
+- If the chaos experiment is not executed, refer to the [troubleshooting](https://docs.litmuschaos.io/docs/faq-troubleshooting/) section to identify the root cause and fix the issues.
+
+### Watch Chaos progress
+  
+- Monitor the VM Instance status using GCP Cloud SDK:
+
+  `gcloud compute instances describe INSTANCE_NAME --zone=INSTANCE_ZONE`
+
+-  GCP console can also be used to monitor the instance status.
+
```
Comment (uditgaurav) [inline docs/gcp-vm-instance-stop.md:260]: If your experiment supports abort pls add.

```md

### Abort/Restart the ChaosExperiment

- To stop the gcp-vm-instance-stop experiment immediately, either delete the ChaosEngine resource or execute the following command:

  `kubectl patch chaosengine <chaosengine-name> -n <namespace> --type merge --patch '{"spec":{"engineState":"stop"}}'`

- To restart the experiment, either re-apply the ChaosEngine YAML or execute the following command:

  `kubectl patch chaosengine <chaosengine-name> -n <namespace> --type merge --patch '{"spec":{"engineState":"active"}}'`
```

File: docs/gcp-vm-instance-stop.md [hunk 7]
```diff
@@ -0,0 +1,269 @@
...
+### Check Chaos Experiment Result
+
+- Check whether the application is resilient to the gcp-vm-instance-stop, once the experiment (job) is completed. The ChaosResult resource name is derived like this: `<ChaosEngine-Name>-<ChaosExperiment-Name>`.
+
+  `kubectl describe chaosresult gcp-vm-chaos-gcp-vm-instance-stop`
+
+### EC2 Terminate Experiment Demo
```
Comment (uditgaurav) [inline docs/gcp-vm-instance-stop.md]:
```suggestion
### GPC VM Instance Stop Experiment Demo
```

File: docs/gcp-vm-instance-stop.md (hunk 5 above)
Comment (neelanjan00) [reply docs/gcp-vm-instance-stop.md]: fixed

File: docs/gcp-vm-instance-stop.md (hunk 6 above)
Comment (neelanjan00) [reply docs/gcp-vm-instance-stop.md:260]: Added

**Review by ispeakc0de — DISMISSED** [review]

**Review by uditgaurav — DISMISSED** [review]

**Comment by uditgaurav** [comment]
embedmd check is failing can we check that?

**Review by uditgaurav — APPROVED** [review]

**Review by ispeakc0de — APPROVED** [review]

**Pull request merged**